nmostesting/suites/IS0401Test.py

```diff
@@ -37,6 +37,11 @@
 CAPS_REGISTER_KEY = "caps-register"
 GROUPHINT_KEY = "grouphint"
 ASSET_KEY = "asset"
+PRINT_KEY = "label"
+
+def get_resource_representation(resource: dict):
+    key = PRINT_KEY if PRINT_KEY in resource else "id"
+    return resource[key]

```

Comment:
how about both label and id? So we augment the output instead of replacing

```suggestion
    return f"{resource[key]} ({resource['id']})"

```